packages/familie-backend/src/auth/utils.ts

```diff
@@ -4,7 +4,6 @@ import { error, logRequest, LOG_LEVEL } from '../logging';
 import { IApi } from '../typer';
 
 export const tokenSetSelfId = 'self';
```

Comment:
Er dette en default-verdi for å hente nyeste tokenset?

Reply:
Dette er default ja :) Det er token for sin egen app. Alle andre er client id til det systemet man skal spørre på.